kernel/packages/shared/apis/SceneStateStorageController/SceneStateStorageController.ts

```diff
@@ -35,13 +35,16 @@ import { ExplorerIdentity } from 'shared/session/types'
 import { deserializeSceneState, serializeSceneState } from 'scene-system/stateful-scene/SceneStateDefinitionSerializer'
 import { ISceneStateStorageController } from './ISceneStateStorageController'
 import { base64ToBlob } from 'atomicHelpers/base64ToBlob'
+import { getLayoutFromParcels } from './utils'
+import { SceneTransformTranslator } from './SceneTransformTranslator'
 
 declare const globalThis: any
 
 export class SceneStateStorageController extends ExposableAPI implements ISceneStateStorageController {
   private readonly builderApiManager = new BuilderServerAPIManager()
   private parcelIdentity = this.options.getAPIInstance(ParcelIdentity)
   private builderManifest!: BuilderManifest
+  private transformTranslator!: SceneTransformTranslator
```

Comment:
Do we need an instance variable for this? Seems like when it is used, it is used like a local variable

Reply:
Most of the time (when the stateful scene to edit is created) a instance of that variable is created.
But we use that reference while saving the scene for example
here: https://github.com/decentraland/explorer/blob/a5d50a174c4c60e3ae951fcb6583ec17108c082a/kernel/packages/shared/apis/SceneStateStorageController/SceneStateStorageController.ts#L88-L111
and here: https://github.com/decentraland/explorer/blob/a5d50a174c4c60e3ae951fcb6583ec17108c082a/kernel/packages/shared/apis/SceneStateStorageController/SceneStateStorageController.ts#L390-L417
any way, it seems that we could use them as local variable safely in those scenarios as well but I choose not to do that just in case :P